### PR TITLE
fix: ECC locking and initFile

### PR DESCRIPTION
**Pull request by marvinmarnold**
Going to do some testing before converting from draft.

**Why**
Manufacturing has slowed for a number of reasons. This PR addresses ECC locking and the initFile intentionally throwing 500s. 

**How**
Lock ECC usage and ensure initFile always returns something.

**References**
- Closes: https://github.com/NebraLtd/hm-diag/issues/167
- Includes: https://github.com/NebraLtd/hm-diag/issues/198
- Includes: https://github.com/NebraLtd/hm-pyhelper/issues/59
- Includes: https://github.com/NebraLtd/hm-diag/issues/181

Additional work still needed:
- https://github.com/NebraLtd/hm-diag/issues/204
- https://github.com/NebraLtd/Hotspot-Production-Tool/issues/32
- https://github.com/NebraLtd/Hotspot-Production-Tool/issues/33

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ services:
       - RELEASE_BUMPER=foobar
 
   diagnostics:
-    image: nebraltd/hm-diag:d444f93
+    image: nebraltd/hm-diag:f476963
```
Comment (marvinmarnold) [inline docker-compose.yml:61]: This should point to https://github.com/NebraLtd/hm-diag/pull/211 after its merged.

**Review by vpetersson — APPROVED** [review]

**Comment by WayneNebra** [comment]
- Connect to your hotspot via Bluetooth CONNECTED **(RED LED1 FLASHING X 12 TIMES)**
- Load the In-App diagnostics of your hotspot CONNECTED AND RUN FINE
- Remove the Wi-Fi connection **SPINNING CIRCLE IN APP** **UNABLE TO ADD WIFI OR FORGET WIFI**
- Add the Wi-Fi connection **UNABLE TO ADD WIFI OR FORGET WIFI**
- Access the IP diagnostics through your local network YES

Monitor the explorer of your hotspot to make sure there are witnesses/ beacons 12-24 hours after update) WILL CHECK 
Run a discovery mode (Might not work, due to issues on Helium side) APP STATES OFFLINE.

Outcome of testing

Red LED1 error flashing x 12 times, unable to disconnect from WIFI or connect WIFI, also one miner has modem false/true missing from diagnostics page.

**Comment by shawaj** [comment]
@WayneNebra indoor shouldn't show modem option

**Comment by WayneNebra** [comment]
owch...n00b alert, sorry about that.

**Comment by shawaj** [comment]
Also, red LED on the ECC board on the outdoor isn't an error led, it has the same function as the green LED on the indoor

**Comment by WayneNebra** [comment]
Thanks I'm learning all the time! sorry for the rookie mistake, with it being off for the last few nights and then seeing it flashing thought it was the following : **Fast Blinking - There is potentially a fault. Please check diagnostics page.**

**Comment by shawaj** [comment]
> Thanks I'm learning all the time! sorry for the rookie mistake, with it being off for the last few nights and then seeing it flashing thought it was the following : **Fast Blinking - There is potentially a fault. Please check diagnostics page.**

No worries at all! Could well have been that - I am not sure how clear the delineation between fast and slow blink is - and I have also noticed ones with no issues whatsoever exhibiting weird LED behaviour. Better to flag it than not!

**Comment by KevinWassermann94** [comment]
@marvinmarnold @vpetersson @shawaj Great work on this one!
I am happy to sign this off, the only issue I am seeing on the testnet is https://github.com/NebraLtd/hm-pktfwd/issues/67 which should not be a blocker.
This helps a significant amount of units

**Pull request merged**